src/components/List.js

```diff
@@ -107,33 +155,37 @@ const List = ({ token }) => {
       {errorMessage && <Error errorMessage={errorMessage} />}
 
       <ul style={{ listStyleType: 'none' }}>
-        {docs &&
-          docs
-            ?.filter((doc) =>
-              doc?.itemName
+        {sortedList &&
+          sortedList
+            ?.filter((item) =>
+              item?.itemName
                 ?.toLowerCase()
                 ?.includes(searchInput.toLowerCase().trim()),
             )
-            ?.map((doc) => {
+            ?.map((item) => {
               return (
-                <li key={doc.id}>
+                <li
+                  key={item.id}
+                  className={backgroundColor(item)}
+                  aria-label={backgroundColor(item)}
```

Comment:
I think this needs a little more description name like : itemname ready to purchase kind of soon. Or something like that. Something that gives screen reader users a clear message. I know you have that text on the screen now for testing purposes, but eventually I'm guessing you won't.